modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -3274,6 +3274,19 @@
 		"issueKey": "LPD-36261",
 		"to": "fetchColumn"
 	},
+	{
+		"from": "_groupPermission.contains",
```

Comment:
@rubenssoares-br The from can't be like this, we can't be sure of the name of the client's variables. Because of that, we use the classNames attributes for non-utility classes.